elementor/elementor
# Internal: Repeater - Add max_items property

## PR Checklist


## PR Type
What kind of change does this PR introduce?

## Summary

This PR can be summarized in the following changelog entry:

* Added a max_items property to the repeater to allow limiting the number of fields that can be adde

## Description
An explanation of what is done in this PR

* added the logic to handle limiting the number of fields that can be added to a repeater. This is achieved by adding the button to add new fields and the copy button

## Test instructions
This PR can be tested by following these steps:

* Create a Repeater field specifying the max_items property
* Verify that in the editor when fields are added to a repeater where the max_items property is specified, the number of those fields can't exceed the max_items property. 

## Quality assurance


Fixes #
